packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -122,23 +122,25 @@ export function TimeRangePicker(props: TimeRangePickerProps) {
         </ToolbarButton>
       </Tooltip>
       {isOpen && (
-        <FocusScope contain autoFocus>
-          <section ref={ref} {...overlayProps} {...dialogProps}>
-            <TimePickerContent
-              timeZone={timeZone}
-              fiscalYearStartMonth={fiscalYearStartMonth}
-              value={value}
-              onChange={onChange}
-              quickOptions={quickOptions}
-              history={history}
-              showHistory
-              widthOverride={widthOverride}
-              onChangeTimeZone={onChangeTimeZone}
-              onChangeFiscalYearStartMonth={onChangeFiscalYearStartMonth}
-              hideQuickRanges={hideQuickRanges}
-            />
-          </section>
-        </FocusScope>
+        <div>
```

Comment:
I get errors in tests when FocusScope doesn't have a wrapping div, but im not entirely sure why. This workaround isn't great....

Reply:
could we swap the order of `FocusScope` and `section`? then at least we don't have an extra element 🤔

Reply:
Yup, that works!